implementations/elixir/ockam/ockam_healthcheck/lib/application.ex

```diff
@@ -14,14 +14,52 @@ defmodule Ockam.Healthcheck.Application do
     Supervisor.start_link(children, strategy: :one_for_one, name: __MODULE__)
   end
 
+  def parse_config(config_json) when is_binary(config_json) do
+    case Jason.decode(config_json) do
+      {:ok, targets} when is_list(targets) ->
+        Enum.reduce(targets, {:ok, []}, fn
+          target, {:ok, targets} ->
+            case target do
+              %{
+                "name" => name,
+                "host" => host,
+                "port" => port,
+                "api_worker" => api_worker,
+                "healthcheck_worker" => healthcheck_worker
+              }
+              when is_integer(port) ->
+                {:ok,
+                 [
+                   %{
+                     name: name,
+                     host: host,
+                     port: port,
+                     api_worker: api_worker,
+                     healthcheck_worker: healthcheck_worker
+                   }
+                   | targets
+                 ]}
+
+              other ->
+                {:error, {:invalid_target, other}}
+            end
+
+          _target, {:error, reason} ->
+            {:error, reason}
+        end)
+
+      {:ok, other} ->
+        {:error, {:invalid_config, other}}
+
+      {:error, reason} ->
+        {:error, {:invalid_config, reason}}
+    end
+  end
+
   def healthcheck_schedule() do
     ## TODO: crontab resolution is per minute
     ## maybe we can use some other tool to schedule healthchecks?
```

Comment:
Not sure if I understand this TODO correctly, but with `SchedEx` we can schedule jobs at higher frequencies than 1 minute:

> Building on the support provided by the crontab library, SchedEx supports extended crontabs. Such crontabs have 7 segments instead of the usual 5; one is added to the beginning of the crontab and expresses a seconds value, and one added to the end expresses a year value. As such, it's possible to specify a unique instant down to the second